agent/logging.go

```diff
@@ -172,7 +179,7 @@ func (a *Agent) convertLoggerConfig(tc *taskContext, c *model.LoggerConfig) clie
 			Sender:            taskConfig.Type,
 			SplunkServerURL:   splunkServer,
 			SplunkToken:       splunkToken,
-			Filepath:          filepath.Join(a.opts.WorkingDirectory, taskLogDirectory, taskLogFileName),
+			Filepath:          filepath.Join(path, taskLogFileName),
```

Comment:
there was a use case here where we would put logs in a different directory so that we could potentially change the image configs so that logs showed up on a different device.

Reply:
Could you elaborate on what you'd like to see changed to handle this?

Reply:
I don't know that we need to do, depending on how "taskLogDirectory" (or its replacement?) is configured. It's possible that it will just work but I just want to make sure we think about that.

Reply:
I really don't understand this use case. Currently logs are put in the task's working directory so that they are cleaned up between tasks, so I'm guessing that won't accomplish this

Reply:
The usecase that max is interested in has to do with people who want to put logs in a different directory on a different device so that log activity won't impact capacity for other kinds of IO activity.

Reply:
made this configurable in the same place that you would specify a logger